omod/src/main/webapp/resources/scripts/mergepatients/patientSearchAndSelectWidget.js

```diff
@@ -0,0 +1,524 @@
+function PatientSearchWidget(configuration){
```

Comment:
I suggest doing this in two passes: (1) bring in everything from the emr module as-is. (2) get rid of redundant things.

In this case let's see if we can use our main find patient widget. and if we need to add extra functionality to display extra columns, we can.